homeassistant/components/system_bridge/config_flow.py

```diff
@@ -35,6 +37,90 @@
     }
 )
 
+system_bridge_data = SystemBridgeData()
```

Comment:
I think we should move this variable and the `_async_handle_module` callback inside `_validate_input` to keep it local. It's not meant for consumption outside `_validate_input`.